src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/components/SceneFunctionDetailsPanel/ui/GitHubCallbackView.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/css';
```

Comment:
Looks ok (though autofocus on close looks weird). I think we could make it a bit simpler with:

```
import { Button, EmptyState, Spinner } from '@grafana/ui';
import React from 'react';

export default function GitHubCallbackView() {
  return (
    <EmptyState
      message="Logging in to GitHub..."
      variant="call-to-action"
      button={<Button onClick={() => window.close()}>Close</Button>}
    >
      <Spinner />
    </EmptyState>
  );
}
```

Reply:
Oh sorry, I see it now. You wanted to have it full-screen without the menu and nav bar around. I think it's better than the previous version anyway so we can merge it 👍